### PR TITLE
Cleanup prerequisites section in README.md

### PR DESCRIPTION
**Pull request by mrunalp**
Make it easier to copy/paste and run.

Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

**Comment by runcom** [comment]
LGTM

**Comment by mrunalp** [comment]
@runcom I just pushed another commit to fixup problems in tools.

**Comment by sameo** [comment]
LGTM

**Comment by pruan-rht** [comment]
verified in Fedora25 & Ubuntu (Mint)

**Comment by mrunalp** [comment]
@pruan-rht Thanks for testing this! :)

File: README.md [hunk 1]
```diff
@@ -43,14 +43,18 @@ It is currently in active development in the Kubernetes community through the [d
...
-In order to enable apparmor support you will need to install development files for `libapparmor` on your platform.
-> e.g. `libapparmor-dev` for Ubuntu
+On Fedora/Centos/RHEL:
+```bash
+dnf install -y btrfs-progs-devel device-mapper-devel libseccomp-devel glib2-devel glibc-devel glibc-static  gpgme-devel libassuan-devel libgpg-error-devel pkgconfig libselinux-devel
+```
+
+On Ubuntu/Debian:
+```bash
+apt install btrfs-tools libassuan-dev libc6-dev libdevmapper-dev libglib2.0-dev libgpg-error-dev libgpgme11-dev pkg-config libseccomp-dev libapparmor-dev
```
Comment (runcom) [inline README.md]: wasn't it `apt-get install`? lol I don't remember but I never typed `apt install`

Comment (runcom) [inline README.md]: @mrunalp ^^

Comment (pruan-rht) [inline README.md]: @runcom, `apt install` and `apt-get install` have the same functionality, AFAIK.  `apt install `is the newer format.  Here's the 'official' repsonse from [askubuntu](http://askubuntu.com/questions/445384/what-is-the-difference-between-apt-and-apt-get)...

Comment (runcom) [inline README.md]: oh great, didn't know that! thanks for explaining!

Comment (mrunalp) [inline README.md]: Yeah, they introduced the simpler apt some time back to unify the commands.

**Pull request merged**